toc/rfc/rfc-draft-promoting-or-revoking-wg-leads.md

```diff
@@ -0,0 +1,51 @@
+# Meta
+[meta]: #meta
+- Name: Promoting and Revoking Working Group Leads
+- Start Date: 2023-10-11
+- Author(s): @ameowlia
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: https://github.com/cloudfoundry/community/pull/704
+
+## Summary
+
+Define the process for promoting or removing Tech Leads and Execution Leads for
+Working Groups.
+
+## Problem
+
+Currently there is no defined process for how to promoting or remove Tech Leads
+or Execution Leads for Working Groups. This process should be defined.
+
+## Proposal
+In this proposal the word "Lead" encompasses both the "Tech Lead" and
+"Execution Lead" roles. As currently stated in the
+[ROLES.md](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md)
+document, both Lead roles are sponsored by the TOC. 
+
+### Promotion to Lead
+* If a person meets the criteria to be a Lead as defined in
+  [ROLES.md](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md),
+  they MAY submit a PR adding themselves to the contributors.yml as a Lead.
+
+* The TOC will start a one week final comment period on the PR to collect
+  approvals from the required parties.
+
+* The TOC MUST approve the PR before it can be merged.
+
+* At least one current Lead for the affected Working Group MUST approve the PR
+  before it can be merged.
+
+* Any current approvers for the affected Working Group MAY 'vote' by commenting
+  "+1" or "-1" on the PR. 75% or more of the votes from current Working Group
+  approvers MUST vote "+1" for the promotion in order for the PR to be merged.
```

Comment:
Should there be a required amount of votes: X% of approvers / X votes in total?

Reply:
I didn't want to force voting. It's already hard enough to get multiple people to approve one PR. But I came to this compromise of allowing voting, but not forcing it so that approvers do have a say.

I am definitely open to hearing other opinions though. Do any other readers have thoughts?